### ISSUE TITLE
quarks-spring-cloud-config-client cannot take "profile" as a property which was supported by spring

### ISSUE THREAD

**Issue by gkrao** [quarkusio/quarkus#32097]
### Describe the bug

I have a rest service that accepts values from bootstrap.yml file for the spring-cloud-config-client to connect to our local Config server. Here is how it looks like:
```
spring:
  cloud:  
    config: 
      enabled: true
      uri: ${CONFIG-URII}
      username: ${CONFIG_USER}
      password: ${CONFIG_PASSWORD}
      name: ${APPNAME}
      profile: ${APPNAME}-${APP_PROFILE}
      label: ${CONFIG_LABEL}
      headers:
        access_token: ${CONFIG_TOKEN}
```
Now in the quarks-spring-cloud-config-client, the above properties need to come from the application.yml. 

In the above, the values of the APPNAME will be "my-rest-service" and APP_PROFILE would be "dev" and effectively, the profile passed would be "my-rest-service-dev".

With the properties "name" and "profile"removed in the quarks-spring-cloud-config. I cannot find a way to get the configuration from the config server for this profile "my-rest-service-dev". 

Would anyone please help resolve this issue?

### Output of `java -version`

11

### Quarkus version or git rev

2.16.0-FINAL

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

 mvn -version Apache Maven 3.8.6 (84538c9988a25aec085021c365c560670ad80f63) Maven home: /Users/grao/Documents/maven Java version: 11.0.2, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk-11.0.2.jdk/Contents/Home Default locale: en_US, platform encoding: UTF-8 OS name: "mac os x", version: "10.16", arch: "x86_64", family: "mac"

**Comment by gkrao** [comment]
Hi Georgios,Thank you for taking time to resolve the issue. Yes, I want to be able to use existing settings to connect to our config servers. Adding the property “profile” would support backward compatibility and would definitely fix our problem. Please let me know if you need any other info. Thanks a lotGopal RaoOn Mar 24, 2023, at 7:31 AM, Georgios Andrianakis ***@***.***> wrote:﻿
So essentially you are saying that you need for us to add profile to https://quarkus.io/guides/spring-cloud-config-client#spring-cloud-config-client-configuration-reference, correct?

**Comment by geoand** [comment]
Thanks for confirming.

I'll have a look soon

**Comment by gkrao** [comment]
Kindly let me know when the fix will be available for using. Thank you GopalOn Mar 25, 2023, at 1:42 AM, Georgios Andrianakis ***@***.***> wrote:﻿
Closed #32097 as completed via #32130.

**Comment by geoand** [comment]
It should be available starting with Quarkus 3.0.0.CR1
